doc/en/user/source/data/raster/imagemosaic/configuration.rst

```diff
@@ -74,7 +74,7 @@ The table below describes the various elements in this configuration file.
      - Suggested plugin for reading the image files.
    * - Envelope2D
      - N
-     - Envelope for the mosaic formatted as ```LLX,LLY URX,URY`` (notice the space between the lower left and upper right coordinate pairs).
+     - Envelope for the mosaic formatted as ``LLX,LLY URX,URY`` (notice the space between the lower left and upper right coordinate pairs).
```

Comment:
I *think* the exta backtick is not actually part of the syntax as it should appear in the configuration file.